pipelines/deploy-release/README.md

```diff
@@ -14,10 +14,10 @@ Tekton pipeline to verify Snapshot prior to Deployment
 | enterpriseContractPublicKey | Public key to use for validation by the enterprise contract | Yes | k8s://openshift-pipelines/public-key |
 | verify_ec_task_bundle | The location of the bundle containing the verify-enterprise-contract task | No | - |
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
-| taskGitRevision | The revision in the taskGitUrl repo to be used | Yes | See Note Below |
+| taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
-* Note: The default taskGitRevision value is taken from whatever branch of release-service-catalog the file is in. It is not maintained
-  in the README, but it can be found by looking in the pipeline yaml definition.
+## Changes in 0.13.0
+- taskGitRevision no longer has a default. It will be provided by the operator. See RHTAPREL-790 for details
```

Comment:
Can you maybe add something like "and will always have the same value as the git revision in the PipelineRef definition of the PipelineRun"?

Reply:
Done